source/howto/setup/aws_private_worker.rst

```diff
@@ -68,9 +72,7 @@ Start by creating a policy that we'll attach to the role:
         ]
     }
 
-..
-
-**Create a IAM role**
+**Create an IAM Role**
 
 * Open the AWS Console and navigate to `IAM -> Roles <https://console.aws.amazon.com/iam/home#/roles>`_
 * Create a new role called `ValohaiWorkerRole` 
```

Comment:
This was one place where I got confused when doing the installation as you cannot give this name on the first page but it comes actually only after you have done all the other steps. Should we mention something about this?

Reply:
Oh yeah I forgot about that. Probably because my notes look like Charlie's musical sheet for Nightman 
![image](https://user-images.githubusercontent.com/341344/124610468-36661180-de79-11eb-9c5f-7d00ab262d06.png)